app/src/main/java/org/mozilla/fenix/utils/ClearableEditText.kt

```diff
@@ -13,9 +13,10 @@ import android.view.MotionEvent
 import android.view.MotionEvent.ACTION_UP
 import androidx.appcompat.content.res.AppCompatResources
 import androidx.appcompat.widget.AppCompatEditText
+import mozilla.components.support.ktx.android.content.getColorFromAttr
 import mozilla.components.support.ktx.android.view.putCompoundDrawablesRelativeWithIntrinsicBounds
 import org.mozilla.fenix.R
-import org.mozilla.fenix.ext.getColorFromAttr
+
```

Comment:
Removing this extra newline will fix the lint issue. There should only be one empty line between the imports and the code

```kt
import org.mozilla.fenix.R

/**
```

Reply:
i thought that was this - 

>[task 2020-02-07T10:04:59.667Z] /builds/worker/checkouts/src/app/src/main/java/org/mozilla/fenix/utils/ClearableEditText.kt:20:1: Needless blank line(s)